Explorer/Assets/Scripts/ECS/SceneLifeCycle/IScenesCache.cs

```diff
@@ -21,18 +24,23 @@ public interface IScenesCache
 
         bool TryGetByParcel(Vector2Int parcel, out ISceneFacade sceneFacade);
 
-        void Clear();
+        bool TryGetPortableExperienceBySceneUrn(string sceneUrn, out ISceneFacade sceneFacade);
+
+        void RemovePortableExperienceFacade(string sceneUrn);
+
+        void ClearScenes(bool clearPortableExperiences = false);
     }
 
     public class ScenesCache : IScenesCache
     {
         private readonly Dictionary<Vector2Int, ISceneFacade> scenesByParcels = new (PoolConstants.SCENES_COUNT * 2);
         private readonly HashSet<Vector2Int> nonRealSceneByParcel = new (PoolConstants.SCENES_COUNT * 2);
-
+        private readonly Dictionary<string, ISceneFacade> portableExperienceScenesByUrn = new (PoolConstants.SCENES_COUNT / 2);
```

Comment:
Does it makes sense to assign such high dictionary size? Portable experiences are not related directly to the scene count, im wrong?

Reply:
agree, it should not be related to `SCENES_COUNT` constant

Reply:
Yeah you are right guys, I used it at the start as a generic value, but tbh we can start with a size of 1 or 2. PXs are so rare that we shouldn't have to worry about the impact of changing the dictionary size if more than 2 are ever activated at the same time.